packages/mvp/src/pages/spaces.tsx

```diff
@@ -88,14 +89,17 @@ const Spaces = () => {
           return !isNaN(rate) && rate <= parseFloat(maxPrice)
         })
     }
-    
+    console.log("results", results.length)
```

Comment:
@ken-980  Can we remove this guy, otherwise looks good to me. You can merge afterwards